modules/local/bagel2/graph.nf

```diff
@@ -0,0 +1,66 @@
+process BAGEL2_GRAPH {
+    tag "${meta.treatment}_${meta.reference}"
+    label 'process_single'
+
+    conda "python=3.11.4 pandas=2.0.3 numpy=1.25.1 scikit-learn=1.3.0 click=8.1.6 matplotlib=3.7.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-54e0353146eca1531516863e8235bf7385d76663:c9ff1a9eec871c54cbea815eae778da702623978-0':
+        'biocontainers/mulled-v2-54e0353146eca1531516863e8235bf7385d76663:c9ff1a9eec871c54cbea815eae778da702623978-0' }"
+
+
```

Comment:
couple of times are here 2 empty lines in a row, 1 might be sufficient?

Reply:
```suggestion
```